src/components/templates/Jazzbar/JazzTab/constants.ts

```diff
@@ -1,74 +1,13 @@
 import { Table } from "types/Table";
 
-export const JAZZBAR_TABLES: Table[] = [
-  {
-    capacity: 6,
-    title: "Table 1",
-    reference: "jazzbar-table-1",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 2",
-    reference: "jazzbar-table-2",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 3",
-    reference: "jazzbar-table-3",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 4",
-    reference: "jazzbar-table-4",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 5",
-    reference: "jazzbar-table-5",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 6",
-    reference: "jazzbar-table-6",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 7",
-    reference: "jazzbar-table-7",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 8",
-    reference: "jazzbar-table-8",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 9",
-    reference: "jazzbar-table-9",
-    rows: 2,
-    columns: 3,
-  },
-  {
-    capacity: 6,
-    title: "Table 10",
-    reference: "jazzbar-table-10",
-    rows: 2,
-    columns: 3,
-  },
-];
+const jazzBarTable = (index: number): Table => ({
+  capacity: 6,
+  title: `Table ${index}`,
+  reference: `jazzbar-table-${index}`,
+  rows: 2,
+  columns: 3,
+});
+
+export const JAZZBAR_TABLES: Table[] = new Array(12)
+  .fill(0)
```

Comment:
Why do we need the .fill(0) here if we’re only using the indexes?

Reply:
Array(number) will create an array with empty elements, which are not mappable

Reply:
Yup, fair enough.

I was thinking of how we do it in `src/components/templates/PartyMap/components/Map/Map.tsx`, but seemingly that is slightly different again:

```typescript
Array.from(Array(totalRows))
```

Reply:
Not that it matters for performance here, but I was curious:

https://jsbench.me/

![image](https://user-images.githubusercontent.com/753891/113792941-7a6a4d00-978a-11eb-8729-572bce0a6a79.png)

The method used here (`.fill(0)`) is ~70% slower than using Array.from(Array(12))

Reply:
@0xdevalias 
thanks for the service! I've never heard about it.

I got curious too. :) By the way, do you know why we can have different results? :see_no_evil: 
![image](https://user-images.githubusercontent.com/63313732/113823247-17c38200-9787-11eb-8681-d73bd9a720e8.png)

Reply:
> thanks for the service! I've never heard about it.

No worries. It's neat/useful when i'm trying to figure between a few choices of ways to implement something, take the guesswork out of it and lets me base the decision on the data.

> By the way, do you know why we can have different results? 🙈

😮 huh.. that's interesting. What browser/OS version are you using out of curiosity? I wonder if there is an optimisation or something in mine that isn't in yours, or vice versa.

I'm using `Google Chrome Version 89.0.4389.90 (Official Build) (x86_64)` on `macOS Big Sur 11.2.3 (20D91)`

I tried testing it in Safari, but that benchmark site seemed to not work for me.

This one did though, and suggested your code was faster: https://jsben.ch/RSVLT

![image](https://user-images.githubusercontent.com/753891/113845587-06a36100-97d9-11eb-863e-07b9a8a73989.png)

Whereas that same test on Chrome suggests mine was:

![image](https://user-images.githubusercontent.com/753891/113845721-25095c80-97d9-11eb-8c74-c853a0630ba9.png)

So yeah, must just come down to the different implementations in different browsers and such. For something like this, the 'real world' performance difference for a 'run once' thing like this that it's irrelevant for anything more than intellectual curiosity. But still, it's interesting to see such a simple case show such a 'dramatic' difference between browsers.

Reply:
I tested that on `Mozilla Firefox for Fedora (version 85.0)`.
Yeah, seems it depends on browser mainly. 

Running your test https://jsben.ch/RSVLT

Firefox:
![image](https://user-images.githubusercontent.com/63313732/114020720-90eed200-9878-11eb-857a-58b89e780c3e.png)

Chrome: 
![image](https://user-images.githubusercontent.com/63313732/114020856-b085fa80-9878-11eb-996a-73c186d64227.png)

yeah, of course. Would be useful to choose a faster way for the features that are going to be frequently used.
thanks again for the tools. It's interesting to compare the performance.

Reply:
No worries :)